src/ua/lviv/iot/algo/part1/lab1/Plate.java

```diff
@@ -0,0 +1,47 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+
+public class Plate {
+
+    private static Plate defaultPlate = new Plate();
+    private double diameter;
+    private String material;
+    private String color;
+    private boolean isClean = false;
+    private boolean hasFood = false;
+
+    public static Plate getInstance(){
+        return defaultPlate;
+    }
+
+    public void wash(){
+        isClean = true;
+    }
+
+    public void dirty(){
+        isClean = false;
+    }
+    public void eat(){
```

Comment:
Your plate cannot eat, come up with a better name

Reply:
such method names are provided by the teacher